core/src/main/java/io/lionweb/lioncore/java/model/ClassifierInstance.java

```diff
@@ -4,6 +4,7 @@
 import java.util.List;
 
 public interface ClassifierInstance<T extends Classifier<T>> extends HasFeatureValues {
+  /** Return all the annotations associated to this Node. */
```

Comment:
I would say `ClassifierInstance` as also `AnnotationInstances` (which are not `Nodes`) can be annotated